bot.py

```diff
@@ -1845,6 +1872,87 @@ async def on_guild_channel_update(
                 f"Category {before.id} was updated but is not associated with a role in cache. This could be an error."
             )
 
+@bot.event
+async def on_guild_emojis_update(guild: discord.Guild, before: Sequence[discord.Emoji], after: Sequence[discord.Emoji]):
+    # PLAN:
+    #   - Check if operation is an add/rename/delete
+    #   - Check if operation was done in control server
+    #       - If not, ask control server if synching should be done
+    #           - If yes, sync
+    #           - If no, do nothing
+    #       - If yes, go ahead and synch
+    bot_commands = bot.get_channel(BOT_COMMANDS_ID)
+
+    # Determine if change was made in control server
+    changed_in_hub = guild.id == HUB_SERVER_ID
+
+    # Determine operation and execute
+
+    # Add
+    if len(before) < len(after):
+        emoji = discord.utils.find(lambda e: e not in before, after)
+        if not emoji:
+            Log.error('find() returned None on detected added emoji')
+            return
+        
+        # Automatically sync throughout all guilds if made in control
+        if changed_in_hub:
+            await sync_add(bot=bot, emoji=emoji)
+            
+        # Send View and wait for acceptance or denial
+        else:
+            # TODO: Format based on desired goals
+            await bot_commands.send(
+                f'An emoji, {emoji.name} with the appearence {emoji} has been added. Would you like to sync this change?', 
```

Comment:
It would be great if the prompt included the name of the server that uploaded it